tests/agents/test_ollama_agent.py

```
@@ -14,134 +17,108 @@
-            'type': 'local', 
-            'args': {'model': 'gpt-oss:20b'}
-        }
+        config2 = {"type": "local", "args": {"model": "gpt-oss:20b"}}
         agent2 = create_agent(config2)
         print(f"✓ Factory created {type(agent2).__name__} for 'local' type")
-        
-        return True
+
+        assert agent1 is not None
+        assert agent2 is not None
     except Exception as e:
```

Ruff (0.14.6)

Do not catch blind exception: `Exception`

(BLE001)